core/src/main/kotlin/org/evomaster/core/problem/rest/RestActionBuilderV3.kt

```diff
@@ -110,7 +110,7 @@ object RestActionBuilderV3 {
      * Create an [ObjectGene] based on schema info of a DTO, given in the format
      * "name: {...}"
      */
-    fun createObjectGeneForDTO(name: String, dtoSchema: String) : Gene{
+    fun createObjectGeneForDTO(name: String, dtoSchema: String, referenceTypeName: String?) : Gene{
```

Comment:
add comment to specify what `referenceTypeName` is, and what it is needed for

Reply:
hi, Based on the OpenAPI, the `name` of `ObjectGene` in RestAction is _body_. For the object gene from DTO, I am not sure whether the `name` of DTO can be always recognized as `refType`. please see the TODO at line 170 in `AbstractRestFitness.kt`.  if it is always true, I might remove refereneTypeName for `createObjectGeneForDTO`